runtime/src/main/java/org/corfudb/runtime/view/StreamOptions.java

```diff
@@ -1,35 +1,27 @@
 package org.corfudb.runtime.view;
 
+import lombok.Builder;
+import lombok.Getter;
+
 /**
  * Created by maithem on 6/20/17.
  */
+@Builder
 public class StreamOptions {
```

Comment:
Can we add some comments as to understand the difference between ReadOptions and StreamOptions, or should the readOptions be part of the StreamOptions?

Reply:
done.